lib/game.js

```diff
@@ -41,11 +41,10 @@ function gameLoop(){
     GamePlay.endGameSequence();
     return true;
   }
-  var newWindup = GamePlay.levelUp(game.dino, game.fruits, game.windups, game.canvas, game.bubbles);
-  if (newWindup) {
-    game.windups = newWindup;
-    GamePlay.nextLevel();
-    // somehow clear out bubbles
+  var newWindups = GamePlay.levelUp(game.dino, game.fruits, game.windups, game.canvas, game.bubbles);
+  if (newWindups) {
+    game.windups = newWindups;
```

Comment:
The transition between levels is a bit strange, so this line has to remain in here for functionality. The windup positions are reset in game_play.js line 116.